src/components/layout/PetCard.css

```diff
@@ -7,7 +7,7 @@
   padding-top: 10px;
   -webkit-box-shadow: 0px 10px 13px -7px #000000,
     5px 5px 15px 5px rgba(0, 0, 0, 0);
-  width: 20rem;
+  width: 30rem;
```

Comment:
lets not change this css maybe just create a new one specifically for the user story cards

Reply:
solved!